autosklearn/automl.py

```diff
@@ -1402,13 +1422,13 @@ def predict(self, X, batch_size=None, n_jobs=1):
         if self.models_ is None or len(self.models_) == 0 or self.ensemble_ is None:
             self._load_models()
 
-        # If self.ensemble_ is None, it means that ensemble_size is set to zero.
+        # If self.ensemble_ is None, it means that ensemble_class is None.
```

Comment:
Is this comment true, are there other cases where `ensemble_` will be None? Perhaps not but just curious.

Reply:
I don't think so.